wazuh/wazuh-dashboard-plugins
# Statistics remove pinned agent

## Description
This PR, removes the pinned agent filter in the statistics dashboard
 
## Issues Resolved
- #6672 

## Evidence

### Before

![Statistics_before](https://github.com/wazuh/wazuh-dashboard-plugins/assets/43619595/071f46a2-bbdc-4fa0-9d5a-10c62dc8b151)

### After

![Statistics_after](https://github.com/wazuh/wazuh-dashboard-plugins/assets/43619595/c70af7be-e441-4a4c-90a1-670f703e6be0)

### Test
- Go to a module and pin an agent.
- Having the agent pinned go to `Server management -> Statistics`
- Check that the opensearch request does not contain the pinned agent filter, nor allow agents or hide manager alerts.
- Return to the beginning module or another module and check that the agent is still pinned.

### Check List
